ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -77,5 +77,45 @@
         </nhs-form-group>
     </validate-input>
     <br />
+
+    <h2>Notification Date</h2>
+    <validate-date model="Notification" property="NotificationDate" notification_id="@Model.Notification.NotificationId" inline-template>
+        @{
+            var NotificationDateError = !Model.IsValid("Notification.NotificationDate");
+            var NotificationDateGroupState = NotificationDateError ? Error : Standard;
+        }
+        <nhs-form-group nhs-form-group-type=@NotificationDateGroupState>
+            <nhs-fieldset aria-describedby="notification-date-error" role="group">
+                <span nhs-span-type="ErrorMessage" ref="errorField"
+                        id="notification-date-error" asp-validation-for="Notification.NotificationDate"></span>

```

Comment:
Can we please add use of the `has-error` helper? See Denotify.cshtml for use of this.
It adds 'display: none', to an element and all it's descendants when passed a true e.g. `NotificationDateError` - the intention of this is to hide the rendered text from nhs-span-type, preventing "Error" being read out by SRs.